shm/.clang-format

```diff
@@ -0,0 +1,8 @@
+BasedOnStyle: Chromium
```

Comment:
Nice idea to add .clang-format to use the same by everyone in a team :) Maybe I will implement it in my ;)